tests/unit/test_deeprec_model.py

```diff
@@ -0,0 +1,40 @@
+
+import pytest
+import os
+from reco_utils.recommender.deeprec.deeprec_utils import *
+from reco_utils.recommender.deeprec.models.xDeepFM import *
+from reco_utils.recommender.deeprec.models.dkn import *
+from reco_utils.recommender.deeprec.IO.iterator import *
+from reco_utils.recommender.deeprec.IO.dkn_iterator import *
+
+@pytest.mark.gpu
+@pytest.mark.deeprec
```

Comment:
This is not needed: `@pytest.mark.deeprec`. The CICD system will take directly everything that has the mark gpu and execute it

Reply:
I prefer to add this tag, because for my offline running, I only need to run some specific part of test cases, and offline run the entire test cases are too time consuming

Reply:
ok sounds good